libr/bin/format/dex/dex.c

```diff
@@ -14,6 +14,12 @@ char* r_bin_dex_get_version(RBinDexObj *bin) {
 	return NULL;
 }
 
+// XXX this is never called
```

Comment:
Why not to remove it then, if its never called?

Reply:
because its a not for my future self to fix the memory leaks that have been there for ever

Reply:
ill fix add the callback in all the plugins to free the rbinobj instance and all the related leaks in a separate PR